### DISCUSSION TITLE
Writing to log file

### DISCUSSION THREAD

**Discussion by carrots66** [burningalchemist/sql_exporter#499] [Q&A] [answered]
Hello, I am using the latest Windows version of the exporter but I cannot manage to get it to write actual log files. Have been looking through documentation and code but unable to figure it out. Can anybody please help?

**Comment by burningalchemist** [comment, marked as answer]
Hey @carrots66, have a look at this issue conversation, it might help - https://github.com/burningalchemist/sql_exporter/issues/367

By default, sql_exporter writes logs to `stdout/stderr` (which is straightforward for Linux server installations) and avoids interacting with a filesystem. On Windows it's tricky, so extra steps are required.

**Reply by carrots66** [reply]
Thanks a million! :)
